ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1432,6 +2027,17 @@ absl::Status ConfigTunnelTermTableEntry(ovs_p4rt::OvsP4rtSession* session,
 // Functions with C interfaces
 //----------------------------------------------------------------------
 
+enum ovs_tunnel_type TunnelTypeStrtoEnum(const char* tnl_type) {
+  if (tnl_type) {
+    if (strcmp(tnl_type, "vxlan") == 0) {
+      return OVS_TUNNEL_VXLAN;
+    } else if (strcmp(tnl_type, "geneve") == 0) {
+      return OVS_TUNNEL_GENEVE;
+    }
+  }
+  return OVS_TUNNEL_UNKNOWN;
+}
+
```

Comment:
This file is getting too big -- should consider breaking it into smaller more manageable files in future